asgi_rabbitmq/core.py

```diff
@@ -835,7 +886,8 @@ def __init__(
         else:
             crypter = None
         self.thread = self.Thread(
-            url, prefix, expiry, group_expiry, self.get_capacity, crypter
+            url, prefix, expiry, group_expiry, self.get_capacity,
+            self.non_local_name, crypter,
```

Comment:
`non_local_name` is [the "almost-static" method](https://github.com/django/channels/blob/master/channels/layers.py#L182-L191) in the base class, it could be rewritten with a helper in our class, and the helper could be reused in the protocol, do you think it's useful to pass it through all the classes? Just wondering.
```
def get_non_local_name(name):
   return ...

class RabbitmqChannelLayer(BaseChannelLayer):
    def non_local_name(self, name):
        return get_non_local_name(name)
```

Reply:
i'm assuming it's not static so that it can be overridden in a subclass, so this method leaves that possibility